src/Components/Lifecycle/Lifecycle.tsx

```diff
@@ -40,13 +44,19 @@ const LifecycleFilters = lazy(() => import('../../Components/LifecycleFilters/Li
 const LifecycleTable = lazy(() => import('../../Components/LifecycleTable/LifecycleTable'));
 import { download, generateCsv, mkConfig } from 'export-to-csv';
 import ErrorState from '@patternfly/react-component-groups/dist/dynamic/ErrorState';
-import { formatDate, getLifecycleType, getNewName } from '../../utils/utils';
+import { formatDate, getNewName } from '../../utils/utils';
 import { Filter } from '../../types/Filter';
 
-const DEFAULT_FILTERS = {
+interface ExtendedFilter extends Filter {
+  viewFilter: string;
+}
```

Comment:
Couldn't this be in the `/types/Filter.ts` file as not-required var defined as `viewFilter?: string`?

Reply:
True let me move it